test/addons/openssl-client-cert-engine/test.js

```diff
@@ -0,0 +1,56 @@
+'use strict';
+const common = require('../../common');
+const assert = require('assert');
+const https = require('https');
+const fs = require('fs');
+const path = require('path');
+
+const engine = path.join(__dirname, '/build/Release/testengine.engine');
+
+const agentKey = fs.readFileSync(path.join(common.fixturesDir,
+    '/keys/agent1-key.pem'));
+const agentCert = fs.readFileSync(path.join(common.fixturesDir,
+    '/keys/agent1-cert.pem'));
+const agentCa = fs.readFileSync(path.join(common.fixturesDir,
+    '/keys/ca1-cert.pem'));
```

Comment:
Minor style nit but core uses arguments-line-up style.  I'm not 100% sure if the linter currently enforces it though.